src/content/docs/features/positioner.mdx

```diff
@@ -3,12 +3,136 @@ title: Positioner
+import { Tabs, TabItem } from '@astrojs/starlight/components';
+import CommandTabs from '@components/CommandTabs.astro';
 
 <PluginLinks plugin="positioner" />
 
-<Stub>
-	Based on
-	https://github.com/tauri-apps/plugins-workspace/tree/v2/plugins/positioner
-</Stub>
+Position your windows at well-known locations.
+
+This plugin is a port of [electron-positioner](https://github.com/jenslind/electron-positioner) for Tauri.
+
+## Supported Platforms
+
+- Windows
+- Linux
+- macOS
+
+## Setup
+
+_This plugin requires a Rust version of at least **1.75**_
+
+Install the positioner plugin to get started.
+
+:::note
+If you only intend on moving the window from Rust code, you only need the dependency in `src-tauri/Cargo.toml`, and can remove the plugin registration from `lib.rs` if you choose to setup automatically.
+:::
+
+<Tabs>
+  <TabItem label="Automatic" >
+
+Use your project's package manager to add the dependency:
+
+{ ' ' }
+
+<CommandTabs
+	npm="npm run tauri add positioner"
+	yarn="yarn run tauri add positioner"
+	pnpm="pnpm tauri add positioner"
+	cargo="cargo tauri add positioner"
+/>
+
+  </TabItem>
+  <TabItem label = "Manual">
+
+1. Install the positioner plugin by adding the following to your `Cargo.toml` file:
+
+```toml title="src-tauri/Cargo.toml"
+[dependencies]
+tauri-plugin-positioner = "2.0.0-beta"
+# alternatively with Git:
+tauri-plugin-positioner = { git = "https://github.com/tauri-apps/plugins-workspace", branch = "v2" }
+```
+
+2. Modify `lib.rs` to initialize the plugin:
+
+```rust title="src-tauri/src/lib.rs" ins={3}
+fn run() {
+    tauri::Builder::default()
+        .plugin(tauri_plugin_positioner::init())
+        .run(tauri::generate_context!())
+        .expect("error while running tauri application");
+}
+```
+
+3. Install the JavaScript Guest bindings using your preferred JavaScript package manager:
+
+  <CommandTabs
+	  npm = "npm install @tauri-apps/plugin-positioner"
+	  yarn = "yarn add @tauri-apps/plugin-positioner"
+	  pnpm = "pnpm add @tauri-apps/plugin-positioner"
+	/>
+  </TabItem>
+</Tabs>
+
+Additional setup is required to get tray-relative positions to work:
+
+```rust title="src-tauri/src/lib.rs" ins={4-12}
+fn run() {
+    tauri::Builder::default()
+        .plugin(tauri_plugin_positioner::init())
+		// This is required to get tray-relative positions to work
+        .setup(|app| {
+            TrayIconBuilder::new()
+                .on_tray_icon_event(|app, event| {
+                    tauri_plugin_positioner::on_tray_event(app.app_handle(), &event);
+                })
+                .build(app)?;
+            Ok(())
+        })
+        .run(tauri::generate_context!())
+        .expect("error while running tauri application");
+}
+```
+
+## Usage
+
+The plugin's APIs are available through the JavaScript guest bindings:
+
+```javascript
+import { move_window, Position } from "@tauri-apps/plugin-positioner";
+
+move_window(Position.TopRight);
```

Comment:
Should be `moveWindow`

Reply:
Yes, you're right.
Lemme open another PR to fix plugins-workspace.